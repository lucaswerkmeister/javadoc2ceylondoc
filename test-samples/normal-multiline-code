/**
 * some {@code code} here
 *
 * this tests the parsing of {@code @code} tags.
 * {@code Here's a longer one}
 */
