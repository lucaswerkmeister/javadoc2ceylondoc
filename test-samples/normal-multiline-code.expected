"some `code` here
 
 this tests the parsing of `@code` tags.
 `Here's a longer one`"
